web/frontend/src/components/slam.vue

```diff
@@ -95,6 +98,66 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeMove = async () => {
+  moveClick = false;
+
+  const req = new motionApi.MoveOnMapRequest();
+
+  /*
+   * set request name
+   * here we set the name of the motion service the user is using
+   */
+  req.setName('builtin');
+
+  const value = await fetchSLAMPose(props.name);
+  // set pose in frame
+  const destination = new commonApi.Pose();
+  destination.setX(Math.abs(destinationMarker.x - value.getX()));
+  destination.setY(Math.abs(destinationMarker.z - value.getY()));
+  destination.setZ(value.getZ());
+  destination.setOX(value.getOX());
+  destination.setOY(value.getOY());
+  destination.setOZ(value.getOZ());
+  destination.setTheta(value.getTheta());
+  req.setDestination(destination);
+
+  // set SLAM resource name
+  const slamResourceName = new commonApi.ResourceName();
+  slamResourceName.setNamespace('rdk');
+  slamResourceName.setType('service');
+  slamResourceName.setSubtype('slam');
+  slamResourceName.setName(props.name);
+  req.setSlamServiceName(slamResourceName);
+
+  // set component name
+  const baseResource = filterResources(props.resources, 'rdk', 'component', 'base');
+  // we are assuming there is only one base that is conducting planning
+  const baseResourceName = new commonApi.ResourceName();
```

Comment:
```suggestion
  const baseResourceName = new commonApi.ResourceNames();
```

Reply:
My apologies I linked the wrong line. My suggestion should have been baseResourceName -> baseResourceNames since it looks like a list (in line 146 we ask for baseResource[0])

Reply:
ah, I understand.
I think that it should remain as `const baseResourceName = new commonApi.ResourceName();` since that is the component resource.Name we need for the motion.MoveOnMap call.

On the other hand i think it would make sense to change 
```
const baseResource = filterResources(props.resources, 'rdk', 'component', 'base');
```
into 
```
const baseResources = filterResources(props.resources, 'rdk', 'component', 'base');
```
since `filterResources` returns a list - possibly of length greater than one.

Reply:
works for me!